README.md

```diff
@@ -14,29 +16,60 @@ A component, integrated with FT authentication and user data services, to add co
 - [Contact](#contact)
 - [Licence](#licence)
 
-### Markup
+## Markup
 
-Use the following markup to enable comments:
+These elements will be initialized automatically on the `o.DOMContentReady` event meaning you're unable to defer component initialisation.
+
+### Stream
+
+Use the following markup to enable a comment stream:
 
 ```html
-<div  class="o-comments" data-o-component="o-comments">
+<div class="o-comments"
+	id="comments"
```

Comment:
Origami components can only used ID attributes if they are namespaced with the origami component name as the namespace. This is to avoid ID collisions within a document.
For example, this is an allowed ID attribute.
```suggestion
	id="o-comments-stream"
```

This is the part of the specification which states this requirement --> https://origami.ft.com/spec/v1/markup/#attributes

Reply:
No problem! I have to remember to update this in `next-article`.